src/frame_graph/frame_graph.hpp

```diff
@@ -228,11 +231,26 @@ namespace wr
 
 				if (!m_rt_properties[i].value().m_is_render_window)
 				{
-					render_system.ResizeRenderTarget(&m_render_targets[i], width, height);
+					uint32_t newWidth = width;
+					uint32_t newHeight = height;
+
+					if (m_rt_properties[i].value().m_width.Get().has_value())
```

Comment:
assign m_properties and m_width to a variable so your not constantly calling get and value.
Also you can do the is more than > 0 check in the same if statement. nested if statements kill readability quickly.

Reply:
If the rt properties do not have a value and i try to check it against a value, it crashes the engine. Therefore I have to do them in separate if statements.